### ISSUE TITLE
Bitbucket usage

### ISSUE THREAD

**Issue by sonecabr** [linkyard/concourse-helm-resource#47]
Is possible to use a private ropository for charts?

**Comment by dodizzle** [comment]
Same question here.
I can clone a private repo using the git source but not sure how to include it in a put.

**Comment by msiegenthaler** [comment]
Yes, you can do that, you just need to `get` the git repo first (using the git-resource) and use that file (e.g. `myrepo/chart/my-chart.tgz`) in the `put` of the helm resource.

**Comment by sfrique** [comment]
@msiegenthaler helm support the use of local folders instead of `.tgz` do you think that is possible with this plugin?
Not sure if make sense to support the package command to create the tgz and then use the tgz to deploy.

**Comment by msiegenthaler** [comment]
we did not add it because:

* we could not tell apart a directory from a chart name (i.e. stable/mysql could also be a directory)
* you'd have to do a `helm dependency build` first for it to work. Altough we probably could integrate that.

**Comment by sfrique** [comment]
@msiegenthaler isn't possible to use a flag? lets say a boolean `local_chart`
Any how, dependency build isn't required if local path contains every chart necessary. 
I have no statics, but I suppose people that use local chart, have everything local.

**Comment by msiegenthaler** [comment]
I guess we could just check whether the path (file) exists and switch on that. I'll see when I find time to implement that. (and of course we love PRs)

**Comment by sfrique** [comment]
@msiegenthaler I went on another direction and created my own image.
I really just need some simple stuff, so I am not even using resources, only tasks for now. Still verifying if concourse will be a good fit! 
Sorry to not be able to to the PR, I would do it for sure if I had the time.

**Comment by sfrique** [comment]
Great, I didn't actually test or use anything yet.
